.github/stale.yml

```
@@ -1,4 +1,24 @@
-daysUntilStale: 60
-daysUntilClose: 7
-onlyLabels:
-  - needs-info
+pulls:
+  daysUntilStale: 60
+  daysUntilClose: 14
+  onlyLabels: []
+  exemptLabels: []
+  markComment: >
+    This pull request has been automatically marked as stale because it has not had
+    recent activity. It will be closed if no further activity occurs. You may re-open
+    it if you need more time. Thank you for your contributions.
```

`Thank you for your contributions.` sounds ironic here. I would either remove, or make it "warmer":

```suggestion
    it if you need more time. We really appreciate your contribution and we are sorry that this hasn't been completed.
```

---

sgtm, but I would move this to the Close comment instead.